db/migrate/20240618044707_create_cohort_lifecycles.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CreateCohortLifecycles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :cohort_lifecycles do | t |
+      t.date :cohort, default: nil
+      t.references :user, null: false, foreign_key: true
```

Comment:
We don't current use foreign key constraints in the DB. While this may be a good practice, I'm not sure we should start here and have this be the only use. I would however add `has_one :cohort_lifecycle, dependent: :delete` to the User model so these will get cleaned up if a user deletes their account